.travis.yml

```diff
@@ -47,3 +47,20 @@ jobs:
       script:
         - scripts/export-translations-github.sh
         - scripts/compile-translations.sh
+    - stage: qaTests
+      if: branch IN (master, boly38)
```

Comment:
Nice idea to use the `if` statement directcly in `.travis.yml`, the same could be applied to `scripts/export-translations-github.sh` and `scripts/compile-translations.sh` scripts.